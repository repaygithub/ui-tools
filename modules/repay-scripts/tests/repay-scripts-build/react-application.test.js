const TestSetup = require('../test-setup')

beforeEach(() => jest.setTimeout(1000 * 60 * 5))

describe('@repay/repay-scripts', () => {
  it('should build a React application', async () => {
    const test = new TestSetup('react-application')
    await test.setup()
    await test.exec(`yarn repay-scripts build index.js`)
    let generatedFiles = await test.readdir('dist')
    expect(generatedFiles).toMatchSnapshot()
    expect(await test.readFile('dist/index.html')).toMatchSnapshot()
    // expect(await test.readFile('dist/main.bundle.js')).toMatchSnapshot()
    // expect(await test.readFile('dist/main.bundle.js.map')).toMatchSnapshot()
  })
})
