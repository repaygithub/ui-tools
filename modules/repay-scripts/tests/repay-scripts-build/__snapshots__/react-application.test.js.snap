// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@repay/repay-scripts should build a React application 1`] = `"<div class=\\"app-root\\"><div class=\\"react-root\\">Testing Text</div></div>"`;
